### DISCUSSION TITLE
Can not start the wifibroadcast@gs.service on boot

### DISCUSSION THREAD

**Discussion by kaylorchen** [svpcom/wfb-ng#131] [Questions]
hello,  I encountered a problem that the service cannot be started.
I type **systemctl enable wifibroadcast@gs**.  After the Pi reboot, this service is not running. But I found this service is loaded, but inactivce. 

 ● wifibroadcast@gs.service - WFB profile gs
   Loaded: loaded (/lib/systemd/system/wifibroadcast@.service; enabled; vendor preset: enabled)
   Active: inactive (dead)

**Comment by svpcom** [comment]
journalctl -xu wifibroadcast@gs

**Reply by kaylorchen** [reply]
Thanks for  your reply, I  have solved the problem.
I need to enable two service, but I just enabled the wifibroadcast@gs.service.
Now, I type:
systemctl enable wifibroadcast.service
systemctl enable wifibroadcast@gs.service
the service can work now.
thanks again.
